### PR TITLE
Add spec text of add and mul

### PR DESCRIPTION
**Pull request by huningxin**
This PR is to add spec texts of element-wise add and multiply according to [r01](https://www.w3.org/2020/04/30-webmachinelearning-minutes.html#r01) of WebML CG Teleconference – 30 April 2020.

> RESOLUTION: Add elementwise add, elementwise multiply, concatenation, and reshape ops to the WebNN API

***
<a href="https://pr-preview.s3.amazonaws.com/huningxin/webnn/pull/54.html" title="Last updated on May 14, 2020, 2:03 AM UTC (7f8b1b5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/webnn/54/928572e...huningxin:7f8b1b5.html" title="Last updated on May 14, 2020, 2:03 AM UTC (7f8b1b5)">Diff</a>

### CODE REVIEW

**Comment by huningxin** [comment]
Thanks @gramalingam for review! Any comments @wchao1115 and @anssiko ?

File: index.bs [hunk 1]
```diff
@@ -328,12 +341,25 @@ partial interface NeuralNetworkContext {
...
+<div algorithm=mul>
+    **Arguments:**
+        - *a*: an {{Operand}}. The first input tensor.
+        - *b*: an {{Operand}}. The second input tensor.
+
+    **Returns:** an {{Operand}}. The output tensor that contains the result of
+    element-wise binary multiplication of the two input tensors.
+
+    Computes the element-wise binary multiplication of the two input tensors.
+    The element-wise binary multiplication will be broadcasted accordingly by
```
Comment (anssiko) [inline index.bs:357]: according to [[!numpy-broadcasting-rule]].

Comment (huningxin) [reply index.bs]: Thanks @anssiko . Fixed it in new commit.

**Comment by huningxin** [comment]
@wchao1115 , I fixed the conflicts of this PR. Please take a look. Thanks!

**Comment by huningxin** [comment]
Thanks all for review. I am going to merge it.

**Pull request merged**